src/main/webapp/app/shared/user-settings/vcs-access-tokens-settings/vcs-access-tokens-settings.html

```diff
@@ -0,0 +1,82 @@
+<h1 jhiTranslate="artemisApp.userSettings.vcsAccessTokenSettings">
+    <!--VCS access token Settings-->
+</h1>
+@if (currentUser) {
+<div class="list-group d-block">
```

Comment:
Please fix the indentions, we indent the elements inside @if one more level

Reply:
Still not fixed?

Reply:
Ok I think that happens due to the pre-commit hook, it just removes the indentation again, if I add it

Reply:
Yep that's what's happening: https://github.com/ls1intum/Artemis/pull/9182/commits/48e81be8723f65f9e149e9874dfd95959602764f here only the newline is kept, all the indentations are gone

Reply:
/home/nom/Pictures/Screenshots/Screenshot from 2024-08-04 18-11-52.png